website/models.py

```diff
@@ -155,6 +156,20 @@ def domain_name(self):
     def get_absolute_url(self):
         return "/domain/" + self.name
 
+    def get_twitter_account(self, name):
```

Comment:
The function both gets/sets twitter account

Reply:
yess because that would much more faster to fetch from database then everytime validating the url and the process. It can be one time and set and just that return